admin-site/src/router/index.ts

```diff
@@ -13,6 +13,12 @@ const routes = [
         // this generates a separate chunk (about.[hash].js) for this route
         // which is lazy-loaded when the route is visited.
         component: () => import(/* webpackChunkName: "home" */ '@/views/Home.vue'),
+
+      },
+      {
+        path: '',
```

Comment:
lets make the path here `/dashboard` so it viewable under localhost:3000/dashboard